libraries/octree/src/Octree.cpp

```diff
@@ -1902,7 +1902,7 @@ bool Octree::readSVOFromStream(unsigned long streamLength, QDataStream& inputStr
     if (getWantSVOfileVersions()) {
 
         // read just enough of the file to parse the header...
-        const unsigned long HEADER_LENGTH = sizeof(PacketType::Value) + sizeof(PacketVersion);
+        const unsigned long HEADER_LENGTH = sizeof(PacketType) + sizeof(PacketVersion);
```

Comment:
need to make this so that it reads/writes the old size so we don't have problems loading old files